src/lib/djerba/plugins/pwgs/analysis/plugin.py

```diff
@@ -54,28 +56,36 @@ def extract(self, config):
         reads_detected = self.preprocess_vaf(config[self.identifier][pc.VAF_FILE])
-        pwgs_base64 = self.write_pwgs_plot(config[self.identifier][pc.HBC_FILE], 
+        pwgs_base64 = self.write_pwgs_plot(config[self.identifier][pc.HBC_FILE],
                                            config[self.identifier][pc.VAF_FILE],
-                                           output_dir = self.workspace.print_location())
+                                           output_dir=self.workspace.print_location())
         self.logger.info("PWGS ANALYSIS: Finished preprocessing files")
-        data = self.get_starting_plugin_data(wrapper, self.PLUGIN_VERSION)       
-        results =  {
-                pc.CTDNA_OUTCOME: mrdetect_results[pc.CTDNA_OUTCOME],
-                pc.SIGNIFICANCE: mrdetect_results[pc.SIGNIFICANCE],
-                pc.TUMOUR_FRACTION_READS: float('%.1E' % Decimal( reads_detected*100 / hbc_results[pc.READS_CHECKED] )),
-                pc.SITES_CHECKED: hbc_results[pc.SITES_CHECKED],
-                pc.READS_CHECKED: hbc_results[pc.READS_CHECKED],
-                pc.SITES_DETECTED: hbc_results[pc.SITES_DETECTED],
-                pc.READS_DETECTED: reads_detected,
-                pc.PVALUE: mrdetect_results[pc.PVALUE],
-                pc.COHORT_N: hbc_results[pc.COHORT_N],
-                'pwgs_base64': pwgs_base64,
-                'files': {
-                    'results_file': config[self.identifier][pc.RESULTS_FILE],
-                    'hbc_results': config[self.identifier][pc.HBC_FILE],
-                    'vaf_results': config[self.identifier][pc.VAF_FILE]
-                }
+        data = self.get_starting_plugin_data(wrapper, self.PLUGIN_VERSION)
+        f_config = configparser.ConfigParser() #Initializing the configparser
+        workspace_dir = self.workspace.get_work_dir()
+        f_config_path = workspace_dir + "/full_config.ini"
+        f_config.read(f_config_path)
+        results = {
+            pc.ASSAY: "plasma Whole Genome Sequencing (pWGS) - 30X (v1.0)",
+            pc.STUDY: f_config["pwgs.case_overview"]["study_title"],
+            pc.PRIMARY_CANCER: f_config["pwgs.case_overview"]["primary_cancer"],
+            pc.CTDNA_OUTCOME: mrdetect_results[pc.CTDNA_OUTCOME],
+            pc.SIGNIFICANCE: mrdetect_results[pc.SIGNIFICANCE],
+            pc.TUMOUR_FRACTION_READS: float('%.1E' % Decimal(reads_detected * 100 / hbc_results[pc.READS_CHECKED])),
+            pc.SITES_CHECKED: hbc_results[pc.SITES_CHECKED],
+            pc.READS_CHECKED: hbc_results[pc.READS_CHECKED],
+            pc.SITES_DETECTED: hbc_results[pc.SITES_DETECTED],
+            pc.READS_DETECTED: reads_detected,
+            pc.PVALUE: mrdetect_results[pc.PVALUE],
+            pc.DATASET_DETECTION_CUTOFF: math.ceil(mrdetect_results[pc.DATASET_DETECTION_CUTOFF]),
+            pc.COHORT_N: hbc_results[pc.COHORT_N],
+            'pwgs_base64': pwgs_base64,
+            'files': {
```

Comment:
Instead of using `config[self.identifier][pc.RESULTS_FILE]`, it's preferred to create a `config_wrapper` object which has many convenience methods for reading Djerba config -- see other plugins for examples.

Reply:
This was marked as "resolved" without fixing the issue, can you explain please? I'm happy to advise on `config_wrapper` usage if needed.

Reply:
Hi Iain , I removed this line 
https://github.com/oicr-gsi/djerba/pull/374/files#diff-db81cc7997a69da0de63924b3a63219b921f2fd3cd0d81cdac1f64fe413f23abL74
Doesnt that address your comment ? Let me know if i misunderstood !

Reply:
Clarified on Slack -- I'm looking forward to the fixes :-)

Reply:
I made some changes
Let me know your thoughts ! Thanks !

Reply:
Much better, thank you.